src/Microsoft.SqlTools.ServiceLayer/Migration/MigrationService.cs

```diff
@@ -248,5 +252,57 @@ public void Dispose()
                 disposed = true;
             }
         }
+
+        [DllImport("advapi32.dll", SetLastError = true, CharSet = CharSet.Unicode)]
+        public static extern bool LogonUser(String lpszUsername, String lpszDomain, String lpszPassword,
+       int dwLogonType, int dwLogonProvider, out SafeAccessTokenHandle phToken);
+
+        internal async Task HandleValidateWindowsAccountRequest(
+            ValidateWindowsAccountRequestParams parameters,
+            RequestContext<ValidateWindowsAccountResult> requestContext)
+        {
+            var domainUserRegex = new Regex(@"^[A-Za-z0-9\\\._-]{7,}$");
+            // Checking if the username string is in 'domain\name' format
+            if (!domainUserRegex.Match(parameters.Username).Success)
+            {
+                await requestContext.SendResult(new ValidateWindowsAccountResult()
```

Comment:
SendError is generally preferred - which will cause an exception to be thrown with the error message